schema/schema.graphql

```diff
@@ -2290,6 +2291,7 @@ enum ExternalServiceKind {
     PAGURE
     PERFORCE
     PHABRICATOR
+    PYTHONPACKAGES
```

Comment:
Q: How is this change + the change adding GERRIT above related to the rest of this patch? Did those changes sneak in through rebasing?

Reply:
This GraphQL file hasn't been updated in a while, so syncing it pulled in a lot of unrelated changes.